string keyword          # Order keyword, e.g. TRANSPORT or MOVE
string priority         # The level of priority for this order; LOW, MEDIUM, HIGH, CRITICAL
string[] order_args     # List of strings containing the order keyword and its arguments.
---
string error_status      # Response status, if order process succeeded(0) or failed(1).
string error_report     # Explaining text of the error in case there was one. 